src/components/flat-table/flat-table-row/__internal__/flat-table-row-draggable.d.ts

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+
+export interface FlatTableRowDraggableProps {
+  /** Array of FlatTableRow. */
+  children: React.ReactNode;
+  /** ID for use in drag and drop functionality */
+  id?: number | string;
+  /** function to find an item in the list of draggable items */
+  findItem: () => object;
```

Comment:
`findItem` and `moveItem` should be optional props (or you need to make them required in the js proptypes - whichever way around is correct